test/java/clojure/lang/UtilTest.java

```diff
@@ -0,0 +1,227 @@
+package clojure.lang;
+
+import com.sun.org.apache.xpath.internal.operations.String;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import java.math.BigInteger;
+import java.util.HashMap;
+
+public class UtilTest {
+
+  @Rule public final ExpectedException thrown = ExpectedException.none();
+
+  @Test
+  public void testClassOf() {
+    Assert.assertNull(Util.classOf(null));
+    Assert.assertEquals(Util.classOf(0), Integer.class);
+  }
+
+  @Test
+  public void testCompare() {
+    Assert.assertEquals(1, Util.compare(false, null));
+    Assert.assertEquals(0, Util.compare(1, 1));
+    Assert.assertEquals(-1, Util.compare(null, 0));
+    Assert.assertEquals(
```

Comment:
break the line after (0,

Reply:
https://google.github.io/styleguide/javaguide.html#s4.5-line-wrapping I was hoping this would help but this only talks about symbols. You already comply with `A comma (,) stays attached to the token that precedes it.`

I think the only statement that holds true is `The primary goal for line wrapping is to have clear code, not necessarily code that fits in the smallest number of lines.`

IMO even though it looks slightly odd I would say its _clearer_ like this
```
Assert.assertEquals(0, 
          Util.compare(new Integer(1), new Integer(1)));
  }
```